examples/routing_service/routing_service_struct_array_transf/cpp/CMakeLists.txt

```
@@ -130,4 +145,14 @@ set_target_properties(SensorDataSubscriber
     PROPERTIES
         CXX_STANDARD 11
         CXX_STANDARD_REQUIRED ON
-)
+        DEBUG_POSTFIX "d"
+        LIBRARY_OUTPUT_DIRECTORY "${output_dir}"
+        LIBRARY_OUTPUT_DIRECTORY_RELEASE "${output_dir}"
+        LIBRARY_OUTPUT_DIRECTORY_DEBUG "${output_dir}"
+        ARCHIVE_OUTPUT_DIRECTORY "${output_dir}"
+        ARCHIVE_OUTPUT_DIRECTORY_RELEASE "${output_dir}"
+        ARCHIVE_OUTPUT_DIRECTORY_DEBUG "${output_dir}"
+        RUNTIME_OUTPUT_DIRECTORY "${output_dir}"
+        RUNTIME_OUTPUT_DIRECTORY_RELEASE "${output_dir}"
+        RUNTIME_OUTPUT_DIRECTORY_DEBUG "${output_dir}"
+)
```

```suggestion
)

```
Please, add an endline!

---

Done.